configs/default/FLMD-DRONIUSF7.config

```diff
@@ -0,0 +1,111 @@
+# Betaflight / STM32F7X2 (S7X2) 4.1.5 Mar 16 2020 / 05:21:46 (d4e74e39c) MSP API: 1.42
+
+board_name DRONIUSF7
+manufacturer_id FLMD
+
+# resources
+resource BEEPER 1 C13
+resource MOTOR 1 A09
+resource MOTOR 2 A08
+resource MOTOR 3 C08
+resource MOTOR 4 C09
+resource PPM 1 B07
+resource LED_STRIP 1 A15
+resource SERIAL_TX 1 B06
+resource SERIAL_TX 2 A02
+resource SERIAL_TX 3 C10
+resource SERIAL_TX 4 A00
+resource SERIAL_TX 5 C12
+resource SERIAL_TX 6 C06
+resource SERIAL_RX 1 B07
+resource SERIAL_RX 2 A03
+resource SERIAL_RX 3 C11
+resource SERIAL_RX 4 A01
+resource SERIAL_RX 5 D02
+resource SERIAL_RX 6 C07
+resource I2C_SCL 1 B08
+resource I2C_SDA 1 B09
+resource LED 1 C14
+resource SPI_SCK 1 A05
+resource SPI_SCK 2 B13
+resource SPI_SCK 3 B03
+resource SPI_MISO 1 A06
+resource SPI_MISO 2 B14
+resource SPI_MISO 3 B04
+resource SPI_MOSI 1 A07
+resource SPI_MOSI 2 B15
+resource SPI_MOSI 3 B05
+resource ADC_BATT 1 C01
+resource ADC_CURR 1 C00
+resource PINIO 1 C02
+resource FLASH_CS 1 B02
+resource OSD_CS 1 B12
+resource GYRO_EXTI 1 B10
+resource GYRO_CS 1 B00
+resource USB_DETECT 1 A10
+
+# timer
+timer B07 AF2
+# pin B07: TIM4 CH2 (AF2)
+timer A09 AF1
+# pin A09: TIM1 CH2 (AF1)
+timer A08 AF1
+# pin A08: TIM1 CH1 (AF1)
+timer C08 AF3
+# pin C08: TIM8 CH3 (AF3)
+timer C09 AF3
+# pin C09: TIM8 CH4 (AF3)
+timer A15 AF1
+# pin A15: TIM2 CH1 (AF1)
+
+# dma
+dma ADC 1 0
+# ADC 1: DMA2 Stream 0 Channel 0
+dma pin B07 0
+# pin B07: DMA1 Stream 3 Channel 2
+dma pin A09 0
+# pin A09: DMA2 Stream 6 Channel 0
+dma pin A08 0
+# pin A08: DMA2 Stream 6 Channel 0
+dma pin C08 0
+# pin C08: DMA2 Stream 2 Channel 0
+dma pin C09 0
+# pin C09: DMA2 Stream 7 Channel 7
+dma pin A15 0
+# pin A15: DMA1 Stream 5 Channel 3
+
+# feature
+feature -RX_PARALLEL_PWM
+feature RX_SERIAL
+feature TELEMETRY
+feature LED_STRIP
+feature OSD
+
+# serial
+serial 0 64 115200 57600 0 115200
+serial 2 2048 115200 57600 0 115200
+
+# master
+set mag_bustype = I2C
+set mag_i2c_device = 1
+set mag_hardware = NONE
```

Comment:
There's no reason to disable the magnetometer. All this does is make it hard for someone if that want to add an external magnetometer later.